src/regions.ts

```diff
@@ -1,35 +1,35 @@
-import { SelectableValue } from '@grafana/data';
+import { type SelectableValue } from '@grafana/data';
 
-export const standardRegions = [
-  'ap-east-1',
-  'ap-northeast-1',
-  'ap-northeast-2',
-  'ap-northeast-3',
+// see https://docs.aws.amazon.com/general/latest/gr/iot-sitewise.html#iot-sitewise_region-sdk
+// order based on order in documentation link
+export const supportedRegions = [
+  'us-east-2',
+  'us-east-1',
+  'us-west-2',
   'ap-south-1',
+  'ap-northeast-2',
   'ap-southeast-1',
   'ap-southeast-2',
+  'ap-northeast-1',
   'ca-central-1',
-  'cn-north-1',
```

Comment:
according to https://www.aws-services.info/iotsitewise.html cn-north-1 is supported

Reply:
Fixed!